src/AspNet.Security.OAuth.Gitter/AspNet.Security.OAuth.Gitter.csproj

```diff
@@ -3,21 +3,25 @@
   <Import Project="..\..\build\packages.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
   </PropertyGroup>
 
   <PropertyGroup>
     <Description>ASP.NET Core security middleware enabling Gitter authentication.</Description>
-    <Authors>Maxime Roussin-Bélanger</Authors>
+    <Authors>Hayden Hancock</Authors>
     <PackageTags>aspnetcore;authentication;gitter;oauth;security</PackageTags>
+    <Company>Hayden Hancock</Company>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="..\..\shared\AspNet.Security.OAuth.Extensions\*.cs" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="JetBrains.Annotations" Version="$(JetBrainsVersion)" PrivateAssets="All" />
+    <Compile Remove="..\..\shared\AspNet.Security.OAuth.Extensions\ClaimsExtensions.cs" />
```

Comment:
This doesn't look right at all. Take a look at the other providers and make that consistent, please.